src/assets/styles/constants/main.scss

```diff
@@ -0,0 +1,9 @@
+$font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell,
+  'Open Sans', 'Helvetica Neue', sans-serif;
+$font-size: 18px;
+
+$dark-font-color: #333333;
+$light-font-color: #ffffff;
+
+$primary-bg-color: #f0f5ff;
```

Comment:
can not understand where u will use this primary and secondary, i see u used it in header, maybe better to name header-primary, header-secondary...

Reply:
To my mind overall structure like that is totally fine

Reply:
I moved the colors into separate variables and now use them in the background variables